app/views/catalog/_embedded_mirador3.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if document.manifest_url.present? %>
   <% manifest_url = document.manifest_url.starts_with?('/') ? root_url + document.manifest_url.slice(1..-1) : document.manifest_url %>
   <%= content_tag :iframe, '',
-        src: "#{Settings.iiif_embed.url}?#{{ url: manifest_url }.to_query}",
+        src: "#{Settings.iiif_embed.url}?#{{ url: manifest_url, canvas_id: choose_canvas_id(block) || params[:canvas_id], iiif_initial_viewer_config: choose_initial_viewer_config(block) || params[:iiif_initial_viewer_config]}.to_query}",
```

Comment:
This is a lot of code for an erb file.  Can this be turned into a ViewComponent, so that some of this can be plain ruby methods?

Reply:
I think that is outside the scope of this very large PR. Can move this into a helper method but I think moving this into a ViewComponent should be it's own PR. I think if we are going to move this into a ViewComponent we should also move https://github.com/sul-dlss/exhibits/blob/dff44d8436cec0e9d950f4bf501b1228a03b9c4a/app/views/catalog/_oembed_default.html.erb into a ViewComponent. So I think that should be it's own PR.